satpy/etc/readers/mirs_l2_nc.yaml

```diff
@@ -0,0 +1,62 @@
+reader:
+  description: NetCDF Reader for the Microwave Integrated Retrieval System Level 2 swath products
+  name: mirs_l2_nc
+  short_name: MiRS Level 2 NetCDF4
+  long_name: MiRS Level 2 Swath Product Reader (NetCDF4)
+  reader: !!python/name:satpy.readers.yaml_reader.FileYAMLReader
+  sensors: [amsu-mhs, atms, ssmis, gmi]
+
+file_types:
+  mirs_atms:
+    file_reader: !!python/name:satpy.readers.mirs_l2_nc.MIRSHandler
+    file_patterns:
+      - 'NPR-MIRS-IMG_v{version}_{platform_shortname}_s{start_time:%Y%m%d%H%M%S}{extra_num1}_e{end_time:%Y%m%d%H%M%S}{extra_num2}_c{creation_time:%Y%m%d%H%M%S}{extra_num3}.nc'
+      - 'NPR-MIRS-SND_v{version}_{platform_shortname}_s{start_time:%Y%m%d%H%M%S}{extra_num1}_e{end_time:%Y%m%d%H%M%S}{extra_num2}_c{creation_time:%Y%m%d%H%M%S}{extra_num3}.nc'
```

Comment:
I don't think this file pattern is needed as this reader can't actually use these files if I'm not mistaken. It at least shouldn't be needed for the related Polar2Grid work.